README.rst

```diff
@@ -25,7 +25,7 @@ This project is available on PyPI, to install it:
 
   pip install thoth-glyph
   
-Features & Usage
+Features
 =================================
```

Comment:
I think some tools will produce wrong output as `=` line is not padded with the heading.